enterprise_access/apps/content_assignments/tasks.py

```diff
@@ -131,3 +132,81 @@ def send_cancel_email_for_pending_assignment(cancelled_assignment_uuid):
         learner_content_assignment_action.traceback = exc
         learner_content_assignment_action.save()
         raise
+
+
+def _get_course_partners(course_data):
+    """
+    Returns a list of course partner data for subsidy requests given a course dictionary.
+    """
+    owners = course_data.get('owners') or []
+    return [{'uuid': owner.get('uuid'), 'name': owner.get('name')} for owner in owners]
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_email_for_new_assignment(new_assignment_uuid):
+    """
+    Send email via braze for new assignment
+
+    Args:
+        new_assignment_uuid: (string) the new assignment uuid
+    """
+    learner_content_assignment_model = apps.get_model('content_assignments.LearnerContentAssignment')
+    subsidy_policy_model = apps.get_model('subsidy_access_policy.SubsidyAccessPolicy')
+
+    try:
+        assignment = learner_content_assignment_model.objects.get(uuid=new_assignment_uuid)
+    except learner_content_assignment_model.DoesNotExist:
+        logger.warning(f'request with uuid: {new_assignment_uuid} does not exist.')
+        return
```

Comment:
Nit: I know it's unlikely, but we should `raise` something here instead of returning.